webapp/src/pages/OnboardingSignup.js

```diff
@@ -0,0 +1,136 @@
+import useToggle from "../shared/react/useToggle";
+import React from "react";
+import Button from "react-bootstrap/Button";
+import Col from "react-bootstrap/Col";
+import Form from "react-bootstrap/Form";
+import Row from "react-bootstrap/Row";
+
+// import { useNavigate } from "react-router";
+
+function OnboardingSignup() {
+  // const navigate = useNavigate();
+  const validated = useToggle(false);
+
+  const handleFormSubmit = (event) => {
+    const form = event.currentTarget;
+    if (form.checkValidity() === false) {
+      event.preventDefault();
+      event.stopPropagation();
+    }
+
+    validated.turnOn();
+    // TODO: should call onboarding api then navigate to dashboard
+    // navigate("/dashboard");
+  };
+  return (
+    <div className="main-container">
+      <Row>
+        <Col>
+          <h2>Member Onboarding</h2>
+          <p className="text-muted small">
+            To join our platform, you are required to enter your address to verify your
+            eligibility for membership.
+          </p>
+          <Form noValidate validated={validated.isOn} onSubmit={handleFormSubmit}>
+            <Form.Group className="mb-3" controlId="name">
+              <Form.Label>Name</Form.Label>
+              <Form.Control
+                // name="name"
+                placeholder="Full Name"
+                required
+              />
+            </Form.Group>
+            <Form.Group className="mb-3" controlId="address1">
+              <Form.Label>Address 1</Form.Label>
+              <Form.Control
+                // name="address"
+                placeholder="e.g. 123 Main Street"
+                required
+              />
+            </Form.Group>
+            <Form.Group className="mb-3" controlId="address2">
+              <Form.Label>Address 2</Form.Label>
+              <Form.Control
+              // name="address2"
+              />
+            </Form.Group>
+            <Row className="mb-3">
+              <Form.Group as={Col} md="6" controlId="cityInput">
+                <Form.Label>City</Form.Label>
+                <Form.Control type="text" placeholder="City" required />
+              </Form.Group>
+              <Form.Group as={Col} controlId="stateInput">
+                <Form.Label>State</Form.Label>
+                <Form.Select defaultValue="" required>
+                  <option disabled value="">
+                    Choose state...
+                  </option>
+                  <option value="AL">Alabama</option>
```

Comment:
Return these from the backend (you can fake an API call for now that just returns `Promise.resolve({states: [{value: 'OR', label: 'Oregon'}]}`). We'll drive these from the DB since most installs will not be national.